services/builder/configProcessor.go

```diff
@@ -92,20 +94,20 @@ func configProcessor(c pb.ContextBoxServiceClient, wg *sync.WaitGroup) error {
 				}
 
 				if ctx, err = buildCluster(ctx); err != nil {
-					log.Error().Msgf("failed to build cluster %s project %s: %s", clusterName, config.Name, err)
+					log.Error().Msgf("Failed to build cluster %s project %s : %v", clusterName, config.Name, err)
```

Comment:
You kept `failed to destroy cluster` here and changed  the previous one to `Error while to destroy(ing)` . Might be wrong but I think we logs should keep the logs in a consistent in form.